src/scripts/activate-user.ts

```diff
@@ -15,6 +15,20 @@ type ActivatableUser = User & {
   readonly status: okta.UserStatus.DEPROVISIONED | okta.UserStatus.STAGED;
 };
 
+/**
+ * Command to activate a user.
+ */
+type ActivateUserCommand = {
```

Comment:
**Issue:** this change does not belong in this PR.

Reply:
reverted change in commit: ca190ab7fd90d8f7902431613ae995edf9f3a57f